packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -507,13 +519,14 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
 		);
 
 		itExpects(
-			"Requesting tombstoned datastores succeeds for legacy document given gcTombtoneGeneration option is defined",
+			"Requesting tombstoned datastores succeeds for legacy document given gcTombstoneGeneration option is defined",
 			[
 				// Interactive client's request that succeeds
 				{
-					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
-					category: "generic",
```

Comment:
Why delete the category?

Reply:
These events looks weird to me. Some of them are not thrown as errors but are still part of the itExpects block. I would rather move those kind of validations into the test itself rather than implictly validating it via test infra.

Reply:
Ok.  AFAIK it's valid to put extra logs in here beyond the errors that are logged. Obv the test would still pass without them being listed here, but when they are listed the test will fail if they don't show up in the logs.